### PR TITLE
Update Screenshots under Rate Limiting

### PR DESCRIPTION
**Pull request by AyeshW**
## Purpose
Update screenshots of pages under `en/docs/design/rate-limiting`
Updated pages are:

- introducing-throttling-use-cases.md
- adding-new-throttling-policies.md
- setting-throttling-limits.md
- access-control.md

### CODE REVIEW

File: en/docs/design/rate-limiting/introducing-throttling-use-cases.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The API Gateway architecture model, which solves the API management problem, com
 -   The applications that consume the APIs in the API Gateway
 -   The users of the applications
 
-![]({{base_path}}/assets/img/learn/throttling-types.png)
+[![]({{base_path}}/assets/img/learn/throttling-types.png)]({{base_path}}/assets/img/learn/throttling-types.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/introducing-throttling-use-cases.md:17]:
```suggestion
[![throttling types]({{base_path}}/assets/img/learn/throttling-types.png)]({{base_path}}/assets/img/learn/throttling-types.png)
```

File: en/docs/design/rate-limiting/introducing-throttling-use-cases.md [hunk 2]
```diff
@@ -60,27 +60,27 @@ You can control/restrict access to your API or its selected resources for a give
...
+[![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)
 
 ##### HTTP request headers
 
 Advanced policies allow you to apply limits to APIs by filtering requests based on HTTP headers. For example, assume you need to apply a special limit for JSON requests. To do that, you can filter JSON messages by using a policy that inspects the HTTP request headers and checks if the `Content-Type` header is `application/json` and apply a special limit for those requests while allowing a default value for the rest.
 
 Here is a sample for configuring a header condition by considering the "Content-Type" header.
 
-![]({{base_path}}/assets/img/learn/new-header-condition-regex.png)
+[![]({{base_path}}/assets/img/learn/new-header-condition-regex.png)]({{base_path}}/assets/img/learn/new-header-condition-regex.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/introducing-throttling-use-cases.md:71]:
```suggestion
[![New header condition regex]({{base_path}}/assets/img/learn/new-header-condition-regex.png)]({{base_path}}/assets/img/learn/new-header-condition-regex.png)

```

**Review by Mariangela — APPROVED** [review]
Looks like there are multiple ALT tags that are missing can you pls add it. I added two ALT tags as suggestions but there are more.

File: en/docs/design/rate-limiting/introducing-throttling-use-cases.md (hunk 2 above)
Comment (AyeshW) [reply en/docs/design/rate-limiting/introducing-throttling-use-cases.md:71]: I will add

Comment (AyeshW) [reply en/docs/design/rate-limiting/introducing-throttling-use-cases.md]: I have added missing ALT tags

File: en/docs/design/rate-limiting/setting-throttling-limits.md [hunk 3]
```diff
@@ -85,13 +85,13 @@ Advanced throttling policies are applied when you are Publishing an API. It can
 
 API level policies can be engaged via the resources section of an API in the Publisher portal by selecting **API Level** under **Rate limitting level** as shown below.
 
-![]({{base_path}}/assets/img/learn/api-level-advanced-policy.png)
+[![Api level advanced policy]({{base_path}}/assets/img/learn/api-level-advanced-policy.png)]({{base_path}}/assets/img/learn/api-level-advanced-policy.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/setting-throttling-limits.md:88]:
```suggestion
[![API level advanced policy]({{base_path}}/assets/img/learn/api-level-advanced-policy.png)]({{base_path}}/assets/img/learn/api-level-advanced-policy.png)
```

File: en/docs/design/rate-limiting/introducing-throttling-use-cases.md [hunk 4]
```diff
@@ -93,7 +93,7 @@ When an application developer subscribes their application to an API, they selec
 The following diagram shows how throttle policies are applied at different levels.
 
 
-![]({{base_path}}/assets/img/learn/throttling-levels.png)
+[![throttling levels]({{base_path}}/assets/img/learn/throttling-levels.png)]({{base_path}}/assets/img/learn/throttling-levels.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/introducing-throttling-use-cases.md:96]:
```suggestion
[![Throttling levels]({{base_path}}/assets/img/learn/throttling-levels.png)]({{base_path}}/assets/img/learn/throttling-levels.png)
```

File: en/docs/design/rate-limiting/introducing-throttling-use-cases.md [hunk 5]
```diff
@@ -60,27 +60,27 @@ You can control/restrict access to your API or its selected resources for a give
...
 The following image shows an example for configuring JWT claim condition by considering the version of the API (http://wso2.org/claims/subscribe).
 
-![]({{base_path}}/assets/img/learn/new-jwt-condition-regex.png)
+[![new jwt condition regex]({{base_path}}/assets/img/learn/new-jwt-condition-regex.png)]({{base_path}}/assets/img/learn/new-jwt-condition-regex.png)
 ##### Query parameters
 
 Filtering based on query parameters almost always apply to HTTP GET requests when doing search type of operations. For example, if you have a search API with `category` as a query parameter, you can have different limits for searching different categories.
 
-![]({{base_path}}/assets/img/learn/new-advanced-throttling-query-conidtion.png)
+[![new advanced throttling query condition]({{base_path}}/assets/img/learn/new-advanced-throttling-query-conidtion.png)]({{base_path}}/assets/img/learn/new-advanced-throttling-query-conidtion.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/introducing-throttling-use-cases.md:83]:
```suggestion
[![New advanced throttling query condition]({{base_path}}/assets/img/learn/new-advanced-throttling-query-conidtion.png)]({{base_path}}/assets/img/learn/new-advanced-throttling-query-conidtion.png)
```

File: en/docs/design/rate-limiting/introducing-throttling-use-cases.md [hunk 6]
```diff
@@ -60,27 +60,27 @@ You can control/restrict access to your API or its selected resources for a give
...
-![]({{base_path}}/assets/img/learn/new-header-condition-regex.png)
+[![new header condition regex]({{base_path}}/assets/img/learn/new-header-condition-regex.png)]({{base_path}}/assets/img/learn/new-header-condition-regex.png)
 ##### JWT claims
 
 A JWT claim contains meta information of an API request. It can include application details, API details, user claims, etc. Advanced throttling policies based on JWT claims allow you to filter requests by JWT claim values and apply limits for requests. For example, if you need to allow special limits for users in a specific user role, you can create an advanced policy that checks for a particular regular expression on the role claim of the user and apply special limits for the ones that match.
 
 The following image shows an example for configuring JWT claim condition by considering the version of the API (http://wso2.org/claims/subscribe).
 
-![]({{base_path}}/assets/img/learn/new-jwt-condition-regex.png)
+[![new jwt condition regex]({{base_path}}/assets/img/learn/new-jwt-condition-regex.png)]({{base_path}}/assets/img/learn/new-jwt-condition-regex.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/introducing-throttling-use-cases.md:78]:
```suggestion
[![New JWT condition regex]({{base_path}}/assets/img/learn/new-jwt-condition-regex.png)]({{base_path}}/assets/img/learn/new-jwt-condition-regex.png)
```

File: en/docs/design/rate-limiting/introducing-throttling-use-cases.md [hunk 7]
```diff
@@ -60,27 +60,27 @@ You can control/restrict access to your API or its selected resources for a give
...
+[![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)
 
 ##### HTTP request headers
 
 Advanced policies allow you to apply limits to APIs by filtering requests based on HTTP headers. For example, assume you need to apply a special limit for JSON requests. To do that, you can filter JSON messages by using a policy that inspects the HTTP request headers and checks if the `Content-Type` header is `application/json` and apply a special limit for those requests while allowing a default value for the rest.
 
 Here is a sample for configuring a header condition by considering the "Content-Type" header.
 
-![]({{base_path}}/assets/img/learn/new-header-condition-regex.png)
+[![new header condition regex]({{base_path}}/assets/img/learn/new-header-condition-regex.png)]({{base_path}}/assets/img/learn/new-header-condition-regex.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/introducing-throttling-use-cases.md:71]:
```suggestion
[![New header condition regex]({{base_path}}/assets/img/learn/new-header-condition-regex.png)]({{base_path}}/assets/img/learn/new-header-condition-regex.png)
```

File: en/docs/design/rate-limiting/introducing-throttling-use-cases.md [hunk 8]
```diff
@@ -14,7 +14,7 @@ The API Gateway architecture model, which solves the API management problem, com
 -   The applications that consume the APIs in the API Gateway
 -   The users of the applications
 
-![]({{base_path}}/assets/img/learn/throttling-types.png)
+[![throttling types]({{base_path}}/assets/img/learn/throttling-types.png)]({{base_path}}/assets/img/learn/throttling-types.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/introducing-throttling-use-cases.md:17]:
```suggestion
[![Throttling types]({{base_path}}/assets/img/learn/throttling-types.png)]({{base_path}}/assets/img/learn/throttling-types.png)
```

File: en/docs/design/rate-limiting/access-control.md [hunk 9]
```diff
@@ -102,10 +102,10 @@ As described above you can deny requests for APIs, by Applications, to IP Addres
...
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/select-context-deny.png)
+    [![Select context deny]({{base_path}}/assets/img/learn/select-context-deny.png)]({{base_path}}/assets/img/learn/select-context-deny.png)
 
 4.  Click **Deny.**
 5.  Now login to API Developer Portal using the URL `https://localhost:9443/devportal` and invoke the API.
     You will see the following response.
 
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/denied-api.png)
\ No newline at end of file
+    [![Denied api]({{base_path}}/assets/img/learn/denied-api.png)]({{base_path}}/assets/img/learn/denied-api.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/access-control.md:111]:
```suggestion
    [![Denied API]({{base_path}}/assets/img/learn/denied-api.png)]({{base_path}}/assets/img/learn/denied-api.png)
```

File: en/docs/design/rate-limiting/access-control.md [hunk 10]
```diff
@@ -82,17 +82,17 @@ To deny a request,
...
 
-![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/select-denied-condition.png)
+[![Select-denied-condition]({{base_path}}/assets/img/learn/select-denied-condition.png)]({{base_path}}/assets/img/learn/select-denied-condition.png)
 <div class="admonition info">
     <p class="admonition-title">Note</p>
 <p>You can temporary switch on/off the denied condition by enabling/disabling the <b>Condition status</b> that is auto enabled when a denied condition is created. </p>
 </div>
 
-![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/denied-condition-status.png)
+[![Denied-condition-status]({{base_path}}/assets/img/learn/denied-condition-status.png)]({{base_path}}/assets/img/learn/denied-condition-status.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/access-control.md:95]:
```suggestion
[![Denied condition status]({{base_path}}/assets/img/learn/denied-condition-status.png)]({{base_path}}/assets/img/learn/denied-condition-status.png)
```

File: en/docs/design/rate-limiting/access-control.md [hunk 11]
```diff
@@ -82,17 +82,17 @@ To deny a request,
 1.  Log in to the Admin Portal using the URL `https://localhost:9443/admin` and your admin credentials.
 2.  Click **Deny Policies** under the **Rate Limiting Policies** section and click **Add Policy**.
 
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/add-denied-policy.png)
+    [![Add-denied-policy]({{base_path}}/assets/img/learn/add-denied-policy.png)]({{base_path}}/assets/img/learn/add-denied-policy.png)
 
 Select the item to deny, enter a value and click **Deny**.
 
-![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/select-denied-condition.png)
+[![Select-denied-condition]({{base_path}}/assets/img/learn/select-denied-condition.png)]({{base_path}}/assets/img/learn/select-denied-condition.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/access-control.md:89]:
```suggestion
[![Select denied condition]({{base_path}}/assets/img/learn/select-denied-condition.png)]({{base_path}}/assets/img/learn/select-denied-condition.png)
```

File: en/docs/design/rate-limiting/access-control.md [hunk 12]
```diff
@@ -82,17 +82,17 @@ To deny a request,
 1.  Log in to the Admin Portal using the URL `https://localhost:9443/admin` and your admin credentials.
 2.  Click **Deny Policies** under the **Rate Limiting Policies** section and click **Add Policy**.
 
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/add-denied-policy.png)
+    [![Add-denied-policy]({{base_path}}/assets/img/learn/add-denied-policy.png)]({{base_path}}/assets/img/learn/add-denied-policy.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/access-control.md:85]:
```suggestion
    [![Add denied policy]({{base_path}}/assets/img/learn/add-denied-policy.png)]({{base_path}}/assets/img/learn/add-denied-policy.png)
```

File: en/docs/design/rate-limiting/access-control.md [hunk 13]
```diff
@@ -56,7 +56,7 @@ When requests to an API are to be permitted from a set of known/trusted IPs only
 1.  Login to API Publisher https://:9443/publisher.
 2.  Select the API and go to the <b>Resources</b> menu.
 3.  Enable **API level** under **Operational Configuration** and select the newly created Throttling policy.
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/apply-allowlist-to-api.png)
+    [![Apply-allowlist-to-api]({{base_path}}/assets/img/learn/apply-allowlist-to-api.png)]({{base_path}}/assets/img/learn/apply-allowlist-to-api.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/access-control.md:59]:
```suggestion
    [![Apply allowlist to API]({{base_path}}/assets/img/learn/apply-allowlist-to-api.png)]({{base_path}}/assets/img/learn/apply-allowlist-to-api.png)
```

File: en/docs/design/rate-limiting/access-control.md [hunk 14]
```diff
@@ -35,17 +35,17 @@ When requests to an API are to be permitted from a set of known/trusted IPs only
...
+    [![Save-allow-specific-ip]({{base_path}}/assets/img/learn/save-allow-specific-ip.png)]({{base_path}}/assets/img/learn/save-allow-specific-ip.png)
       
      <div class="admonition info">
     <p class="admonition-title">Note</p>  
     <p>In the above configuration we are allowing a Specific IP address to access an API.
     You can allow a range of IPs as well by selecting **IP Range** for the IP Condition Type in the Conditional Group and specifying the range as follows.</p>
     </div>
 
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/new-allowed-ip-range.png)
+    [![New-allowed-ip-range]({{base_path}}/assets/img/learn/new-allowed-ip-range.png)]({{base_path}}/assets/img/learn/new-allowed-ip-range.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/access-control.md:48]:
```suggestion
    [![New allowed IP range]({{base_path}}/assets/img/learn/new-allowed-ip-range.png)]({{base_path}}/assets/img/learn/new-allowed-ip-range.png)
```

File: en/docs/design/rate-limiting/access-control.md [hunk 15]
```diff
@@ -35,17 +35,17 @@ When requests to an API are to be permitted from a set of known/trusted IPs only
 
     Following is an example configuration.
 
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)
+    [![New-allow-specific-ip]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)
     
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/save-allow-specific-ip.png)
+    [![Save-allow-specific-ip]({{base_path}}/assets/img/learn/save-allow-specific-ip.png)]({{base_path}}/assets/img/learn/save-allow-specific-ip.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/access-control.md:40]:
```suggestion
    [![Save allow specific IP]({{base_path}}/assets/img/learn/save-allow-specific-ip.png)]({{base_path}}/assets/img/learn/save-allow-specific-ip.png)
```

File: en/docs/design/rate-limiting/access-control.md [hunk 16]
```diff
@@ -35,17 +35,17 @@ When requests to an API are to be permitted from a set of known/trusted IPs only
 
     Following is an example configuration.
 
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)
+    [![New-allow-specific-ip]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/access-control.md:38]:
```suggestion
    [![New allow specific IP]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)
```

File: en/docs/design/rate-limiting/access-control.md [hunk 17]
```diff
@@ -18,11 +18,11 @@ When requests to an API are to be permitted from a set of known/trusted IPs only
 2.  Under the **Rate Limiting Policies** tab and navigate to **Advanced Policies.**
 3.  Click Add New Policy to add a new Rate Limit tier.
 
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/allowed-ip-add-policy.png)
+    [![Allowed-ip-add-policy]({{base_path}}/assets/img/learn/allowed-ip-add-policy.png)]({{base_path}}/assets/img/learn/allowed-ip-add-policy.png)
 
 4.  Fill the details as below and click **Add Conditional Group**.
 
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/allowed-ip-add-conditional-group.png)
+    [![Allowed-ip-add-conditional-group]({{base_path}}/assets/img/learn/allowed-ip-add-conditional-group.png)]({{base_path}}/assets/img/learn/allowed-ip-add-conditional-group.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/access-control.md:25]:
```suggestion
    [![Allowed IP add conditional group]({{base_path}}/assets/img/learn/allowed-ip-add-conditional-group.png)]({{base_path}}/assets/img/learn/allowed-ip-add-conditional-group.png)
```

File: en/docs/design/rate-limiting/access-control.md [hunk 18]
```diff
@@ -18,11 +18,11 @@ When requests to an API are to be permitted from a set of known/trusted IPs only
 2.  Under the **Rate Limiting Policies** tab and navigate to **Advanced Policies.**
 3.  Click Add New Policy to add a new Rate Limit tier.
 
-    ![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/allowed-ip-add-policy.png)
+    [![Allowed-ip-add-policy]({{base_path}}/assets/img/learn/allowed-ip-add-policy.png)]({{base_path}}/assets/img/learn/allowed-ip-add-policy.png)
```
Comment (Mariangela) [inline en/docs/design/rate-limiting/access-control.md:21]:
```suggestion
    [![Allowed IP add policy]({{base_path}}/assets/img/learn/allowed-ip-add-policy.png)]({{base_path}}/assets/img/learn/allowed-ip-add-policy.png)
```

**Comment by Mariangela** [comment]
@AyeshW , do not add dashes (-) when adding ALT tags.

**Pull request merged**